### ISSUE TITLE
How to render youtube, twitter etc... in the markdown ?

### ISSUE THREAD

**Issue by hlassiege** [nuxt/content#1046]
Hi,

I noticed that it was possible to add remark plugins in order to enhance the behaviour of nuxt-content.

However I'm not sure how to make it work.

I added the following plugin remark-oembed in basePlugins :

      basePlugins: ['remark-squeeze-paragraphs', 'remark-slug', 'remark-autolink-headings', 'remark-external-links', 'remark-footnotes',
        '@agentofuser/remark-oembed'],

I also tried with _remarkPlugins_ and _plugins_
in a markdown file, I have for example : 

    https://www.youtube.com/watch?v=7Pq-S557XQU

However it seems the rendering don't use the plugin and the link stays unchanged.

Maybe there is another plugin to do that ?
Do you know if there is something specific to make it work ?

**Comment by farnabaz** [comment]
@hlassiege You also need to register your custom plugins under [`markdown.remarkPlugins`](https://content.nuxtjs.org/configuration#markdownremarkplugins) in `nuxt.config`.

```js
  content: {
    liveEdit: false,
    markdown: {
      remarkPlugins: [
        '@agentofuser/remark-oembed'
      ],
      prism: {
        theme: "prism-themes/themes/prism-dracula.css",
      },
    },
  },
```

**Comment by hlassiege** [comment]
Yes I did that but it didn't work. I tried in remarkPlugin, plugins and markdownPlugins as stated in the documentation.
But I found another solution with remark plugins.
Now it's directly transformed on client side with javascript. 
(See https://github.com/hlassiege/eventuallycoding/blob/master/pages/_.vue#L222)

**Comment by farnabaz** [comment]
Seems that  `@agentofuser/remark-oembed` did not update since long time it does not export transformer properly. Consider using alternative plugins like [remark-oembed](https://www.npmjs.com/package/remark-oembed) which is clone of `@agentofuser/remark-oembed`

! Using `remark-oembed` you might need to add some styles, it does not provide any styling

**Comment by TechWatching** [comment]
Any up-to-date alternative? What are you using on your websites?

**Comment by hlassiege** [comment]
> Any up-to-date alternative? What are you using on your websites?

Hi, If the question is for me, I use a custom solution as described here : 

https://github.com/nuxt/content/issues/1046#issuecomment-1133830963
